app/app.js

```diff
@@ -40,4 +43,9 @@ let start = async () => {
     }
 };
 
+function timeout(ms) {
+    logger.info(`sleeping for ${ms} ms before strating runner`);
+    return new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
setTimeout get's number as timeout argument according to its signature:

`declare function setTimeout(handler: TimerHandler, timeout?: number, ...arguments: any[]): number;`

and here it is a string. I would parse it in the config

Reply:
done